components/style/themes/default.less

```
@@ -763,6 +763,7 @@
 @drawer-header-padding: 16px 24px;
 @drawer-body-padding: 24px;
 @drawer-bg: @component-background;
+@drawer-footer-padding: 10px 16px;
```

We can use `@modal-footer-padding` as value.

---

There isnt an existing `@modal-footer-padding` variable, can i suggest

```suggestion
@drawer-footer-padding: @modal-footer-padding-vertical @modal-footer-padding-horizontal;
```

---

or `@drawer-footer-padding-vertical: @modal-footer-padding-vertical;`  and `@drawer-footer-padding-horizontal: @modal-footer-padding-horizontal;`